.github/workflows/release.yml

```diff
@@ -0,0 +1,96 @@
+name: Release
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'Release Version'
+        required: true
+        type: string
+
+jobs:
+  build:
+    strategy:
+      matrix:
+        product: [clion-oss-latest-stable, clion-oss-oldest-stable]
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Setup Bazel
+        uses: bazel-contrib/setup-bazel@0.15.0
+        with:
+          bazelisk-cache: true
+          disk-cache: ${{ github.workflow }}
+          repository-cache: true
+
+      - name: Update version.bzl
+        run: |
+          echo "VERSION = '${{ inputs.version }}'" > version.bzl
+
+      - name: Build Plugin Zip
+        run: bazel build //clwb:clwb_bazel_zip --define=ij_product=${{ matrix.product }}
+
+      - name: Rename Plugin Zip
+        run: |
+          bazel build //intellij_platform_sdk:build_name --define=ij_product=${{ matrix.product }}
+          mkdir out
+          mv bazel-bin/clwb/clwb_bazel.zip out/$(cat bazel-bin/intellij_platform_sdk/build_name.txt).zip
+
+      - name: Upload Plugin Zip
+        uses: actions/upload-artifact@v4
+        with:
+          name: ${{ matrix.product }}
+          path: "out/*.zip"
+          retention-days: 1
+
+  gh-release:
+    needs: build
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Download Latest Plugin Zip
+        uses: actions/download-artifact@v4
+        with:
+          name: clion-oss-latest-stable
+
+      - name: Download Oldest Plugin Zip
+        uses: actions/download-artifact@v4
+        with:
+          name: clion-oss-oldest-stable
+
+      - name: Create Release
+        uses: softprops/action-gh-release@v1
+        with:
+          files: "*.zip"
+          tag_name: ${{ inputs.version }}
+          name: "CLwB Release ${{ inputs.version }}"
+          draft: false
+          prerelease: false
+          generate_release_notes: true
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+  jb-release:
+    needs: build
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Download Latest Plugin Zip
+        uses: actions/download-artifact@v4
+        with:
+          name: clion-oss-latest-stable
+
+      - name: Download Oldest Plugin Zip
+        uses: actions/download-artifact@v4
+        with:
+          name: clion-oss-oldest-stable
+
+      - name: Upload Releases
+        run: |
+          find . -type f -name "*.zip" -exec curl -i --header "Authorization: Bearer ${{ secrets.MARKETPLACE_TOKEN }}" -F pluginId=9554 -F file=@{} -F isHidden=true -F channel=stable https://plugins.jetbrains.com/plugin/uploadPlugin \; 1> /dev/null
```

Comment:
maybe passing errors to `/dev/null` is unnecessary here as otherwise how we know what is an actual error?

Reply:
Shouldn't 1 be stdout? Because the web request returns the entire plugin description and it was hard to see what's going on. That's why I only wanna keep stderr. 

Reply:
True, my mistake. But then can we use silent curl flag or -o /dev/null?